nix/.stack.nix/io-sim-classes.nix

```diff
@@ -0,0 +1,20 @@
+{ system, compiler, flags, pkgs, hsPkgs, pkgconfPkgs, ... }:
+  {
+    flags = {};
+    package = {
+      specVersion = "1.10";
+      identifier = { name = "io-sim-classes"; version = "0.1.0.0"; };
+      license = "NONE";
+      copyright = "2018 IOHK";
+      maintainer = "";
+      author = "Alexander Vieth, Marcin Szamotulski, Duncan Coutts";
```

Comment:
Please also add Karl Knutsson (@karknu) to the authors.

Reply:
Or use `IOHK Engineering Team` as in  `ouroboros-consensus`.

Reply:
Everything in `.stack.nix` (and the `.stack-pkgs.nix` file) is generated from cabal/stack config (via `nix/regenerate.sh`).
Probably best to correct cabal files in another PR.

Reply:
ok, we'll do that in another PR